Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu
+
+# set up timezone
+RUN echo "America/New_York" > /etc/timezone
+
+# install required packages
+RUN apt-get update \
+&& DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    python3 \
+    python3-dev \
+    python3-pip \
+    ffmpeg python3-tk \
+&& rm -rf /var/lib/apt/lists/*
+
+# setup locales and language to utf-8
+ENV LC_ALL C.UTF-8
+ENV LANG C.UTF-8
+
+# create a new directory
+RUN mkdir /MobOff
+
+# copy current directory to /MobOff 
+COPY . /MobOff
```

Comment:
It would be portable and stand alone, if we can do `git clone <>` here, instead of creating a directory & adding files. You might have to install git in the _installation_ step

Reply:
The `Dockerfile` should be portable and self sufficient?

Reply:
By standalone I mean, in this scenario we are expecting the user who is building the Docker image to have the _MobOff_ already downloaded in the same directory as the _Dockerfile_ and we are copying it in to the Docker image. By using `git clone` we need not have any assumptions as such. Less the assumptions, more portable and stand alone would be the application.